main/parse.c

```diff
@@ -3344,6 +3344,7 @@ static bool createTagsWithFallback1 (const langType language,
 			*/
 			setTagFilePosition (&tagfpos);
 			setNumTagsAdded (numTags);
+			writerRescanFailed (numTags);
```

Comment:
@masatake Am I right here to assume that numTags here contains the number of tags added to the file before the current scan (which failed)? This is how I use it in the client and the latest patches in Geany and if it isn't correct, re-parsing won't work correctly.

Reply:
You are right.